### ISSUE TITLE
Annotations should be available for Enum.Case.

### ISSUE THREAD

**Issue by RabugenTom** [krzysztofzablocki/Sourcery#59]
Hi everyone!

As of version 0.4.3, annotations are available for types and variables, but not for cases of enum's.

I think it would be very nice to have access to annotations for enum cases. 

For example, I'm trying to wrap localized strings in some custom `MyLocalizable` structs and starting from
````swift
enum Localizable {
    enum Category {
        /// sourcery comment = "Some comment"
        case someKey = "Default value"
    }
}
````
I would like to generate
````swift
struct Localizable {
    struct Category {
        static let someKey = MyLocalizable(key: "Localizable.Category.someKey", default: "Default value", comment: "Some comment")
    }
}
````
Please note that the enum is not part of the project. It is used only by Sourcery to generate code.
`MyLocalizable` has a `string` var that returns the localized string and I can then simply use `Localizable.Category.someKey.string`. where I need to. The Localizable.strings files can easily be generated from Sourcery too.

With the current version, the best I can do is to use a struct instead of an enum:
````swift
struct Localizable {
    struct Category {
        /// sourcery comment = "Some comment"
        /// sourcery default = "Default value"
        let someKey: String
    }
}
````
Here, I need to use another comment for the default since one doesn't have the rawValue anymore. Furthermore, I need to specify the type of key, even if I'm not using it after. This is not as optimal as the enum case.

An alternative solution would be to give access to literal values of variables when they have some. For example, for the variable `let foo = "bar"` could maybe have a literalValue property returning `"bar"`.

**Comment by krzysztofzablocki** [comment]
Makes total sense to me, I'll enable annotation for cases today, should be very straighforward 👍

**Comment by krzysztofzablocki** [comment]
@RabugenTom available in 0.4.4
